tests/components/vizio/test_media_player.py

```diff
@@ -180,11 +179,15 @@
                 + [
                     app["name"]
                     for app in device_config[CONF_APPS][CONF_ADDITIONAL_CONFIGS]
+                    if app["name"] not in APP_LIST
                 ]
             )
         else:
             list_to_test = list(INPUT_LIST_WITH_APPS + APP_LIST)
 
+        if CONF_ADDITIONAL_CONFIGS in device_config.get(CONF_APPS, {}):
```

Comment:
Having conditional statements in the tests isn't good. It's better to write some extra lines and simplify the test helper (or remove it) than add logic to the tests.

Reply:
OK fair point. I can refactor this but I do this in a couple of places, so it will require a lot of changes to tests and might be better to submit that as a separate PR to clean up tests. Thoughts?

Reply:
Yes, it's ok to do in a separate PR.